protocol/SignalingProtocol.proto

```diff
@@ -304,11 +304,23 @@ message SdkTranscriptItem {
     }
     optional Type type = 6;
     optional bool vocabulary_filter_match = 7;
+    optional double confidence = 8;
+    optional bool stable = 9;
+}
+
+message SdkTranscriptEntity {
+    optional string category = 1;
```

Comment:
Is it possible that an older version of an SDK client receives these messages from the transcription service?

If so, what happens?